mycpp/cppgen_pass.py

```diff
@@ -2687,7 +2688,7 @@ def visit_class_def(self, o: 'mypy.nodes.ClassDef') -> T:
                                 callee.name == '__init__'):
                             base_constructor_args = expr.args
                             #log('ARGS %s', base_constructor_args)
-                            self.def_write(' : %s(', base_class_name)
+                            self.def_write(' : %s(', join_name_cpp(base_class_name, True))
```

Comment:
should call this with strip_package=True (generally we follow the rule that optional args are always named)

Also why does strip_package exist?  It could use a comment

Reply:
Ah, yeah `strip_package` thing is awkward. Apparently we omit the top level directories (e.g. `core`, `builtin`, `ysh`) from the c++ namespaces. Will add a comment